coral/src/app/features/topics/acl-request/schemas/topic-acl-request-shared-fields.ts

```diff
@@ -32,11 +48,19 @@ const acl_ssl = z
     }
   )
   .optional();
-const aclPatternType = z.union([z.literal("LITERAL"), z.literal("PREFIXED")]);
+const aclPatternType = z.union([
+  z.literal("LITERAL", {
+    errorMap: () => ({ message: "Please choose an option." }),
+  }),
+  z.literal("PREFIXED", {
+    errorMap: () => ({ message: "Please choose an option." }),
+  }),
+]);
 const topicname = z.string().min(1, { message: "Please enter a prefix." });
-const environment = z
-  .string()
-  .min(1, { message: "Please select an environment" });
+const environment = z.string({
+  errorMap: () => ({ message: "Please  select an environment." }),
```

Comment:
Extra space between Please and select 👀

Reply:
[0f81784](https://github.com/aiven/klaw/pull/1047/commits/0f817841af5eafdfe1e202769db7f6ec79179798)